lib/consyncful/persisted_item.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 module Consyncful
-  # Takes a mapped item from contentful and applies it to the local storage.
+  ##
+  # Takes a mapped item from contentful and applies it to the local database.
+  #
```

Comment:
I wonder if using more specific wording than applies would be useful? Saying that it create/update/deletes the entry from the database. I forgot it did deletions too until I expanded it and read the code, which made me think that might be useful? 🤷‍♂️